lib/react_client.dart

```diff
@@ -217,6 +218,19 @@ final ReactDartInteropStatics _dartInteropStatics = (() {
     component.transferComponentState();
   }
 
+  void _callSetStateCallbacks(Component component) {
+    component.setStateCallbacks.forEach((callback()) { callback(); });
```

Comment:
is there a reason for the parens to be here?  `forEach((callback())` vs `forEach((callback)`